src/main/java/bio/terra/service/snapshotbuilder/query/tables/Concept.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.service.snapshotbuilder.query.tables;
+
+import bio.terra.service.snapshotbuilder.query.FieldVariable;
+import bio.terra.service.snapshotbuilder.query.SourceVariable;
+import bio.terra.service.snapshotbuilder.query.TablePointer;
+
+public class Concept extends Table {
+
+  public static final String TABLE_NAME = "concept";
+  public static final String DOMAIN_ID = "domain_id";
+  public static final String CONCEPT_NAME = "concept_name";
+  public static final String CONCEPT_ID = "concept_id";
+  public static final String STANDARD_CONCEPT = "standard_concept";
+  public static final String CONCEPT_CODE = "concept_code";
+  private static final TablePointer tablePointer = TablePointer.fromTableName(TABLE_NAME);
```

Comment:
I'm still not sure if this should be a `static` or not. It's immutable so there's no harm in it, but it's also small and there's not much overhead in making a new one each time. But if it is `static`, then it should be renamed to follow the java naming conventions
```suggestion
  private static final TablePointer TABLE_POINTER = TablePointer.fromTableName(TABLE_NAME);
```

or maybe
```suggestion
  private static final TablePointer POINTER = TablePointer.fromTableName(TABLE_NAME);
```